ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -94,7 +95,7 @@ fi
 FCST_LEN_HRS="1"
 FCST_LEN_HRS_SPINUP="1"
 POSTPROC_LEN_HRS="1"
-POSTPROC_LONG_LEN_HRS="2"
+POSTPROC_LONG_LEN_HRS="24"
```

Comment:
I think this is consistent with the configuration. @chunhuazhou please correct me!

Reply:
For the current RRFSE setup, forecast jobs run for the 1st hour for 18Z-23Z. And then 36 hours for 00Z. Can we have the flexibility to run the graphics accordingly? This way we don't have a bunch of graphics tasks even when we only have 1-hour forecast. Or do we only run the graphics for 00Z?

Reply:
I have set it up to run the graphics task for every cycle. We can certainly change that. It's all up to y'all. 

When I ran this test with this config, it looked like the forecast length was set to 24 hours, and queues off this variable. The length of the forecast graphics created is set by FCST_LEN_HRS_CYCLES -- that sets the first item to 24 and the rest to 1 hour, which is where I got 24 here.